src/main.cpp

```diff
@@ -4622,7 +4623,7 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
     }
 
 
-    else if (strCommand == "alert")
+    else if (fAlerts && strCommand == "alert")
```

Comment:
Thanks for the code review, more than a year after the fact.

However this code has been ripped out already together with the alert logic, so it's no longer relevant.